src/analysis/cpu_platform_profiles.md

```diff
@@ -37,10 +39,12 @@ PORTB=io
 ```
 Here, `PINB` is the name and `io` is the type of the port and this will be added as a 
 flag at the offset `0x03`. The type can be `ext_io` if it's an extended IO register, as well. Both will be added 
-as [flags](https://book.rizin.re/basic_commands/flags.html?highlight=flags#flags) and the only difference between 
-them is that they will be added in different [flagspaces](https://book.rizin.re/refcard/intro.html?highlight=flagspa#flagspaces).
+as [flags](https://book.rizin.re/src/basic_commands/flags.html?highlight=flags#flags) and the only difference between 
```

Comment:
These links should be created by the tool instead of hardcoding them. see https://github.com/rizinorg/book/blob/master/src/analysis/signatures.md?plain=1#L24

Reply:
you can use also `../some/dir/file.md`